### PR TITLE
Fix warnings

### PR DESCRIPTION
**Pull request by dl1jbe**
First part of diverse changes to fix GCC 10 stringop warnings.

### CODE REVIEW

File: src/clusterinfo.c [hunk 1]
```diff
@@ -208,12 +208,17 @@ int loadbandmap(void) {
...
 	    g_strlcpy(spotcall, thisline + 26, 6);
 
-	    strncpy(spottime, thisline + 70, 4);	// how old?
-	    spottime[4] = spottime[3];
-	    spottime[3] = spottime[2];
-	    spottime[2] = ':';
-	    spottime[5] = '\0';
-	    spotminutes = 60 * atoi(spottime) + atoi(spottime + 3);
+	    /* read and convert hours and minutes to spotminutes */
+	    spottime[0] = *(thisline + 70);
```
Comment (zcsahok) [inline src/clusterinfo.c:212]: maybe `spottime[x] = thisline[y]` would be more readable

ideally one would need to check if `thisline` has enough content, something like strlen >= 74, on line 207 in the if()

File: src/tlf.h [hunk 2]
```diff
@@ -149,7 +149,7 @@ typedef struct {
     char call[20];
     int countrynr;
     char continent[3];
-    char cqzone[3];
+    unsigned short cqzone;
```
Comment (zcsahok) [inline src/tlf.h:152]: I'd just use a plain `int`, even if wasting some bytes.

**Review by zcsahok — COMMENTED** [review]
yesterday forgot to update test/data.c, will post a PR shortly

File: src/clusterinfo.c (hunk 1 above)
Comment (dl1jbe) [reply src/clusterinfo.c:212]: Right. Good. Idea. Wrt the length I will look into it.

File: src/tlf.h (hunk 2 above)
Comment (dl1jbe) [reply src/tlf.h:152]: yes, we also had countrynr as int

**Comment by N0NB** [comment]
As for scroll_log.c, you may as well match it to the fixes in the other files using `g_strndup()`.

**Comment by dl1jbe** [comment]
No more comments -> will apply now

**Pull request merged**